pkg/vpa/vpa.go

```diff
@@ -338,7 +335,7 @@ func (r Reconciler) getVPAObject(existingVPA *vpav1.VerticalPodAutoscaler, ns *c
 			Name:       vpaName,
 		},
 		UpdatePolicy: &vpav1.PodUpdatePolicy{
-			UpdateMode: vpaUpdateModeForResource(ns),
+			UpdateMode: updateMode,
```

Comment:
```suggestion
			UpdateMode: vpaUpdateMode,
```

updateMode is undeclared. I'm only guessing at the name of the actual value based on what else is in this function. Please have a look and make sure.

Reply:
Whoops, thanks, fixed